src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedInstallIngress.tsx

```diff
@@ -1,16 +1,102 @@
+import { useAuthProvider } from 'Auth/MAPI/MapiAuthProvider';
 import { Box, Heading, Paragraph } from 'grommet';
 import InstallIngressButton from 'MAPI/apps/InstallIngressButton';
-import React from 'react';
+import { usePermissionsForClusters } from 'MAPI/clusters/permissions/usePermissionsForClusters';
+import { hasClusterAppLabel } from 'MAPI/clusters/utils';
+import { Cluster } from 'MAPI/types';
+import { extractErrorMessage, fetchCluster, fetchClusterKey } from 'MAPI/utils';
+import { GenericResponseError } from 'model/clients/GenericResponseError';
+import * as securityv1alpha1 from 'model/services/mapi/securityv1alpha1';
+import React, { useEffect } from 'react';
 import { Breadcrumb } from 'react-breadcrumbs';
 import { useRouteMatch } from 'react-router';
+import useSWR from 'swr';
+import ErrorReporter from 'utils/errors/ErrorReporter';
+import { FlashMessage, messageTTL, messageType } from 'utils/flashMessage';
+import { useHttpClient } from 'utils/hooks/useHttpClient';
+import { useHttpClientFactory } from 'utils/hooks/useHttpClientFactory';
 
 interface IGettingStartedInstallIngressProps {}
 
 const GettingStartedInstallIngress: React.FC<
   React.PropsWithChildren<IGettingStartedInstallIngressProps>
 > = () => {
```

Comment:
Since this component now includes more than just display functionality, what do you think about moving it to `src/components/MAPI/clusters/GettingStarted/` ?

Reply:
Sounds good, makes sense